recipes-support/rauc-hawkbit-updater/rauc-hawkbit-updater_git.bb

```diff
@@ -0,0 +1,37 @@
+SUMMARY = "The RAUC hawkBit updater operates as an interface between the RAUC D-Bus API and the hawkBit DDI API."
```

Comment:
in meta-yogurt exist a recipe with the same name, but different content. only the layer priority decide about the use of the recipe. The rauc-hawkbit-updater do not use versions. we should discuss with mschwan-phytec about the recipe name.

Reply:
We should not recreate this recipe in the first place, but rather use a bbappend to install the link to the config file (which is the only thing we do here anyway).